test/tests/functional/pbs_resv_start_dur_end.py

```diff
@@ -140,3 +146,22 @@ def test_end_wall_resv(self):
 
         a = {'reserve_state': (MATCH_RE, 'RESV_CONFIRMED|2')}
         self.server.expect(RESV, a, id=rid)
+
+    def test_rstat_longterm_resv(self):
+        """
+        Test to submit a reservation, where duration > INT_MAX.
+        Check whether rstat displaying negative number.
+        """
+        now = int(time.time())
+        a = {'Resource_List.select': '1:ncpus=1',
+             'reserve_start': now + 3600,
+             'reserve_end': now + 4294970895}
+        r = Reservation(TEST_USER, attrs=a)
+        rid = self.server.submit(r)
+        a = {'reserve_state': (MATCH_RE, 'RESV_CONFIRMED|2')}
+        self.server.expect(RESV, a, id=rid)
+
+        rstat_cmd = [os.path.join(self.pbs_exec, 'bin', 'pbs_rstat')]
```

Comment:
Please use PTL's status() function to stat the reservation instead of directly invoking pbs_rstat from the test.